x-pack/plugins/ml/public/embeddables/anomaly_swimlane/explorer_swimlane_container.tsx

```diff
@@ -0,0 +1,289 @@
+          const allowedDiff = 20;
+          const diff = Math.abs(curr - prev);
+          return diff < allowedDiff;
+        })
+      ),
+      refresh.pipe(startWith(null)),
+    ])
+      .pipe(
+        debounceTime(500),
+        switchMap(([jobs, input, swimlaneContainerWidth]) => {
+          const {
+            viewBy,
+            swimlaneType: swimlaneTypeInput,
+            limit,
+            timeRange,
+            filters,
+            query,
+          } = input;
+
+          explorerService.setTimeRange(timeRange);
+
+          if (!swimlaneType) {
+            setSwimlaneType(swimlaneTypeInput);
+          }
+
+          const explorerJobs: ExplorerJob[] = jobs.map(job => {
+            const bucketSpan = parseInterval(job.analysis_config.bucket_span);
+            return {
+              id: job.job_id,
+              selected: true,
+              bucketSpanSeconds: bucketSpan!.asSeconds(),
+            };
+          });
+
+          let appliedFilters: any;
+          try {
+            appliedFilters = processFilters(filters, query);
+          } catch (e) {
+            // handle query syntax errors
+            setError(e);
+            return of(undefined);
+          }
+
+          return from(explorerService.loadOverallData(explorerJobs, swimlaneContainerWidth)).pipe(
+            switchMap(overallSwimlaneData => {
+              const { earliest, latest } = overallSwimlaneData;
+
+              if (overallSwimlaneData && swimlaneTypeInput === SWIMLANE_TYPE.VIEW_BY) {
+                return from(
+                  explorerService.loadViewBySwimlane(
+                    [],
+                    { earliest, latest },
+                    explorerJobs,
+                    viewBy!,
+                    limit!,
+                    swimlaneContainerWidth,
+                    appliedFilters
+                  )
+                ).pipe(
+                  map(viewBySwimlaneData => {
+                    return {
+                      ...viewBySwimlaneData!,
+                      earliest,
+                      latest,
+                    };
+                  })
+                );
+              }
+              return of(overallSwimlaneData);
+            })
+          );
+        }),
+        catchError(e => {
+          setError(e.body);
+          return of(undefined);
+        })
+      )
+      .subscribe(data => {
+        if (data !== undefined) {
+          setError(null);
+          setSwimlaneData(data);
+        }
+      });
+
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+
+  const onResize = throttle((e: { width: number; height: number }) => {
+    const labelWidth = 200;
+    setChartWidth(e.width - labelWidth);
+  }, RESIZE_THROTTLE_TIME_MS);
+
+  if (error) {
+    return (
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.ml.swimlaneEmbeddable.errorMessage"
+            defaultMessage="Unable to load the swimlane data"
```

Comment:
Maybe worth adding in `ML` to this message, `Unable to load the ML swimlane data`

Reply:
Check https://github.com/elastic/kibana/pull/64056/commits/5801a72accf0b20097aefba46ac6183b2a5bd1c4